docs/standard/garbage-collection/fundamentals.md

```diff
@@ -210,55 +198,79 @@ The following are threading and performance considerations for server garbage co
 
 - Because multiple garbage collection threads work together, server garbage collection is faster than workstation garbage collection on the same size heap.
 
-- Server garbage collection often has larger size segments. Note, however, that this is only a generalization: segment size is implementation-specific and is subject to change. You should make no assumptions about the size of segments allocated by the garbage collector when tuning your app.
+- Server garbage collection often has larger size segments. However, this is only a generalization: segment size is implementation-specific and is subject to change. Don't make assumptions about the size of segments allocated by the garbage collector when tuning your app.
 
-- Server garbage collection can be resource-intensive. For example, if you have 12 processes running on a computer that has 4 processors, there will be 48 dedicated garbage collection threads if they are all using server garbage collection. In a high memory load situation, if all the processes start doing garbage collection, the garbage collector will have 48 threads to schedule.
+- Server garbage collection can be resource-intensive. For example, if you have 12 processes running on a computer that has 4 processors, there will be 48 dedicated garbage collection threads if they are all using server garbage collection. In a high memory-load situation, if all the processes start doing garbage collection, the garbage collector has 48 threads to schedule.
```

Comment:
>In a high memory-load situation, if all the processes start doing garbage collection, the garbage collector has 48 threads to schedule. [](start = 241, length = 135)

I would say "if all processes happen to be doing GC at the same time they would interfere with each other as there'd be 12 threads scheduled on the same processor. If these processes are active, it wouldnt be a good idea to have them all use server GC"